src/global/components/Logo.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import logo from '../../../public/logo.jpeg';
+import Image from 'next/image';
+import { useMediaQuery } from '@react-hook/media-query';
+export const Logo = () => {
+    const isTabletOrMobile = useMediaQuery('(max-width: 768px)');
+    
+  return (
+   <Image
```

Comment:
What do you think about making the logo clickable? it is not important feature you can do it later